### PR TITLE
[PBW-4071] publishing OO.EVENTS.NON_LINEAR_AD_DISPLAYED when overlay …

### PR DESCRIPTION
**Pull request by ksunika**
…is loaded (VAST)

### CODE REVIEW

**Comment by ksunika** [comment]
should be looked at together with http://github.services.ooyala.net/Platform/mjolnir/pull/516

File: js/controller.js [hunk 1]
```diff
@@ -36,6 +36,7 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       "adOverlayUrl": null,
       "showAdOverlay": false,
       "showAdOverlayCloseButton": false,
+      "adOverlayLoaded": false,
```
Comment (chibzu) [inline js/controller.js]: Trying to think through the use case, do we actually need to store this state or just pass the message along the message bus? I'm not coming up with any reason the skin needs to care about the overlay being loaded or not aside from the check in adOverlay.js .

**Comment by ksunika** [comment]
@chibzu, you are right. I removed the unnecessary state from controller.

**Comment by chibzu** [comment]
Awesome, looks good.

**Pull request merged**